### PR TITLE
More null entity fixes

### PR DESCRIPTION
**Pull request by ghost**
Additional checks for PVS & containers to fix exceptions from null entities.

### CODE REVIEW

File: Robust.Client/GameObjects/Components/Containers/ContainerManagerComponent.cs [hunk 1]
```diff
@@ -144,6 +144,11 @@ public override void HandleComponentState(ComponentState? curState, ComponentSta
                 // Add new entities.
                 foreach (var uid in data.ContainedEntities)
                 {
+                    if (!Owner.EntityManager.TryGetEntity(uid, out var ent))
+                    {
+                        continue;
+                    }
+
                     var entity = Owner.EntityManager.GetEntity(uid);
```
Comment (RemieRichards) [inline Robust.Client/GameObjects/Components/Containers/ContainerManagerComponent.cs]:
```suggestion
                    if (!Owner.EntityManager.TryGetEntity(uid, out var entity))
                    {
                        continue;
                    }
```

File: Robust.Client/GameObjects/Components/Containers/ContainerManagerComponent.cs [hunk 2]
```diff
@@ -144,7 +144,10 @@ public override void HandleComponentState(ComponentState? curState, ComponentSta
                 // Add new entities.
                 foreach (var uid in data.ContainedEntities)
                 {
-                    var entity = Owner.EntityManager.GetEntity(uid);
+                    if (!Owner.EntityManager.TryGetEntity(uid, out var entity))
```
Comment (PJB3005) [inline Robust.Client/GameObjects/Components/Containers/ContainerManagerComponent.cs:147]: This really should never be possible. I would rather a test case for this get made and the root bug fixed than ignoring the error here.

**Review by PJB3005 — CHANGES_REQUESTED** [review]
None of the cases where you added checks *should* ever be capable of having deleted entities in them. You should fix the reason these deleted entities are in here instead of making PVS resilient against them.

**Comment by ghost** [comment]
This shouldn't be necessary anymore actually since I made singulo not delete entities that are within a container, but that's what was causing it. I think the container was getting deleted and then the items within it were getting deleted afterwards, somehow.

**Pull request closed without merging**